Makefile

```diff
@@ -217,7 +217,7 @@ test-extended:
 #
 # Example:
 #   make run
-run: export OS_OUTPUT_BINPATH=$(shell bash -c 'source hack/common.sh; echo $${OS_OUTPUT_BINPATH}')
+run: export OS_OUTPUT_BINPATH=$(shell bash -c 'source hack/lib/init.sh; echo $${OS_OUTPUT_BINPATH}')
 run: export PLATFORM=$(shell bash -c 'source hack/common.sh; os::build::host_platform')
 run: build
 	$(OS_OUTPUT_BINPATH)/$(PLATFORM)/openshift start
```

Comment:
This still doesn't work for because it tries to run `/linux/amd64/openshift start` but my openshift binary is in `/data/src/github.com/openshift/origin/_output/local/bin/linux/amd64/openshift`